app/models/ArticlesViewedSettings.scala

```diff
@@ -9,5 +9,6 @@ case class MaxViews(
 case class ArticlesViewedSettings(
  minViews: Option[Int],
  maxViews: Option[Int],
- periodInWeeks: Int
+ periodInWeeks: Int,
+ tags: List[String]=Nil
```

Comment:
this name doesn't match the SDC model - https://github.com/guardian/support-dotcom-components/pull/1134/files#diff-1bf2b8e1c7e257fc1c135431146989151ac2299f647de0de7b70bdc6ea2d347fR81

Reply:
great spot!Thank you